modules/ROOT/pages/config-workflow.adoc

```diff
@@ -20,5 +20,5 @@ endif::[]
 After completing the steps in this workflow, you can deploy and manage applications, register APIs and use the API Exchange on an Anypoint Platform environment running on your own local network.
 
 [NOTE]
-PCE containers and processes must run as root for port binding and system and kernel configuration. Containers running as root
-are not exposed outside the cluster. All pods that are exposed outside the cluster run as non root.
+Some Docker containers must run as root for port binding and system and kernel configuration. Containers running as root
```

Comment:
@paigehargrave `port binding` and `system and kernel configuration` can ring a security bell in our customers.
I'd just mention the fact that it's how it is now, without an explanation on why. And also add that we are working on moving away from it.

Reply:
Updated this - thank you!